project/core/templates/landing.html

```diff
@@ -159,16 +157,18 @@ <h4 class="center ampersand">&</h4>
 </main>
 
 </script>
+{% endblock backbone_template %}
 
-<body>
+{% block content %}
     {% include "static_nav.html" %}
     <div id="landing"></div>
     {% include "static_footer.html" %}
```

Comment:
Let's move the static navigation and footer to the `base.html`.
```suggestion
    <div id="landing"></div>
```